wbooks/src/app/screens/dashboard/components/Catalog/index.js

```diff
@@ -1,30 +1,32 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 import Book from '../Book';
 import './style.css';
 import PropTypes from 'prop-types'; 
 
 const Catalog = (props) => {
     const bookList = props.books.map((book) => 
-      <Book key={`book_${book.id}`} 
-        image_url={book.image_url} 
+      <Link to={'/books/'+book.id} key={`book_${book.id}`}>
```

Comment:
use the constant defined in routes.js and add the book id

${BOOKS}/${book.id}